ui/lib/apps/ClusterInfo/components/StoreLocation.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useMemo } from 'react'
+import { useClientRequest } from '@lib/utils/useClientRequest'
+import client, { TopologyStoreLocation } from '@lib/client'
+import { Pre } from '@lib/components'
+
+type TreeNode = {
+  name: string
+  value: string
+  children: TreeNode[]
+}
+
+function buildTopology(data: TopologyStoreLocation | undefined) {
```

Comment:
It is easier to do the aggregation in the frontend, so I didn't do it in the backend.

Reply:
I agree